src/common/redux/states/api_key.ts

```diff
@@ -0,0 +1,14 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+export interface IApiKey {
+    key: string,
+    provider: string,
```

Comment:
`provider` is not a string but an enumeration of provider name define as an apiKeys identifier. One an unique key per provider name i presume ?